utils/uris.js

```diff
@@ -2,7 +2,7 @@ import * as base58 from "micro-base58";
 import { v1 as uuid } from 'uuid';
 
 export const appPrefix =
-  process.env.NEXT_PUBLIC_APP_PREFIX || "apps/understory/garden"
+  process.env.NEXT_PUBLIC_APP_PREFIX || "apps/mysilio/garden"
```

Comment:
I don't think we should change this bit. AFAIK, this is only used for the legacy storage paths (since we use `/spaces/...` with GardenKit), and while we haven't changed the env var, it still seems less likely to cause migration issues if we leave things as is until we're ready to remove it entirely.

Reply:
ah that makes sense to me!